app/assets/stylesheets/custom/ows_slide.less

```diff
@@ -3,6 +3,7 @@
 *******************************/
 
 .ows_slide {
+  font-size: 10px;
```

Comment:
Slides don't automatically resize yet, so that's probably necessary to make it fit into the sidebar. But it should be `.sidebar .ows_slide { font-size .... }` so that the TempSlideTestPage isn't affected.